### DISCUSSION TITLE
How to Establish Predecessor-Successor Relationships in FrePPLe to Manage Work Order Sequences?

### DISCUSSION THREAD

**Discussion by salvastp** [frePPLe/frepple#599] [Q&A]
Hello everyone,

Is it possible to establish predecessor and successor relationships between work orders in FrePPLe? I'm looking for a way to manage the sequence of work orders effectively.

Thank you in advance for your responses.

Best regards,

**Comment by jdetaeye** [comment]
Yes, you can dependencies between operations:
- doc https://frepple.com/docs/current/model-reference/operation-dependencies.php 
- live demo model https://frepple.com/docs/current/examples/operation/operation-dependency.php  https://demo.frepple.com/operation-dependency/supplypath/demand/rocket%20for%202023%20Q2/?noautofilter/

**Reply by salvastp** [reply]
I think I haven't explained myself clearly. I'm not looking for relationships between operations but between different production orders that manufacture components that are later assembled in the top-level parent order. These are independent orders but need to be manufactured in a specific sequence.

**Comment by jdetaeye** [comment]
Same answer....    You'll need to define the dependencies between the operations of these manufacturing orders.   You can't define them directly between the manufacturing orders.

We do have some customers where there is a 1-to-1 relationship between an operation and a manufacturing order.    The plans then look very much like a project-plan for a list of tasks.        
A "classic" frepple model has many manufacturing orders on the same operation.   Eg we plan a manufacturing order on the operation for every sales order or forecast bucket. 

I think the project-planning type of model fits your description, and yes, frepple can handle that.

**Reply by salvastp** [reply]
Thank you for your response.

I believe I can structure my data and operations to align with the functionality of FrePPLe. Each group of orders will have its specific operations linked together, creating a simulation of relationships between the production orders. I appreciate your assistance, and I'll proceed to prepare the data and conduct some tests.

Thank you for your help.
